pages.es/common/aapt.md

```diff
@@ -0,0 +1,17 @@
+# aapt
+
+> Herramienta para empaquetado de activos de Android.
+> Compila y empaqueta recursos de una app de Android.
+> Más información: <https://elinux.org/Android_aapt>.
+
+- Lista los archivos contenidos en un archivo APK:
+
+`aapt list {{ruta/a/app.apk}}`
+
+- Muestra la metadata de una app (versión, persmisos, etc.):
+
+`aapt dump badging {{ruta/a/app.apk}}`
+
+- Crea un nuevo archivo APK con archivos de un directorio especificado:
+
+`aapt package -F {{ruta/a/app.apk}} {{ruta/al/directorio}}`
```

Comment:
Shouldn't it be like this?
```suggestion
`aapt list {{ruta/al/app.apk}}`

- Muestra la metadata de una app (versión, persmisos, etc.):

`aapt dump badging {{ruta/al/app.apk}}`

- Crea un nuevo archivo APK con archivos de un directorio especificado:

`aapt package -F {{ruta/al/app.apk}} {{ruta/al/directorio}}`
```

Reply:
no, because the English version use `path/to/app.apk`, in Spanish that's `ruta/a/app.apk` y sounds good in Spanish :+1:

Reply:
:thinking: both are good, but for consistency yep, you're right.